zap/src/main/java/org/zaproxy/zap/extension/alert/AlertAPI.java

```diff
@@ -297,7 +297,9 @@ public JSON toJSON() {
             boolean recurse = this.getParam(params, PARAM_RECURSE, false);
 
             // 0 (RISK_INFO) -> 3 (RISK_HIGH)
-            int[] counts = new int[] {0, 0, 0, 0};
+            int[] riskCounts = new int[] {0, 0, 0, 0};
+            // 0 (CONFIDENCE_FALSE_POSITIVE)
+            int[] confidenceCounts = new int[] {0};
```

Comment:
It could just be `int` and named `falsePositiveCount`. Since the endpoint is about the risks I don't think it's going to return all confidences.